countdown.js

```diff
@@ -1,5 +1,12 @@
+
+function log_seconds(n){
+  console.log(n+"...");
+}
+
 function countdown(seconds){
-  // ...
+  for (var i=0; i<=seconds; i++){
+    window.setTimeout(log_seconds, 1000*i, seconds-i);
```

Comment:
Nice, I didn't even realize that you could pass additional arguments to `setTimeout()`!